### PR TITLE
Translate 1349 words into chinese simplified

### PR DESCRIPTION
**Pull request by hushuilan**
Translate 1349 words into chinese simplified

### CODE REVIEW

File: zh-CN/Economics and Ethics of Private Property Studies in Political Economy and Philosophy/03_p01_ch02_01.md [hunk 1]
```diff
@@ -40,32 +40,32 @@
...
 
-[^1]: Exclusively descriptive analyses of taxation are given, for instance, by Paul Samuelson, *Economics*, 10th ed. (New York: McGraw Hill, 1976), chap. 9; Roger L. Miller, *Economics Today*, 6th ed. (New York: Harper and Row, 1988), chap. 6.
+[^1]：例如，保罗萨缪尔森，*经济学*，第10版给出了对税收的独特描述性分析。 （纽约：麦格劳希尔，1976年），第一章。 9;罗杰L.米勒，*今日经济*，第6版。 （纽约：Harper and Row，1988），第6章。 
 
-[^2]: Jean Baptiste Say, *A Treatise on Political Economy* (New York: Augustus M. Kelley, 1964), pp. 446–47.
+[^2]：Jean Baptiste Say，*政治经济学论著*（纽约：Augustus M. Kelley，1964），第446-47页。
```
Comment (rosatravels) [inline zh-CN/Economics and Ethics of Private Property Studies in Political Economy and Philosophy/03_p01_ch02_01.md]: @huishilan, you can translate the word 'say' there to chinese.

Comment (rosatravels) [inline zh-CN/Economics and Ethics of Private Property Studies in Political Economy and Philosophy/03_p01_ch02_01.md]: @huishilan, you can translate the word 'say' there to chinese.

**Comment by hushuilan** [comment]
Hi, dear @rosatravels . The author's full name should be Jean Baptiste Say. I have corrected the translation error in this place. Thank you.

File: zh-CN/Economics and Ethics of Private Property Studies in Political Economy and Philosophy/03_p01_ch02_01.md (hunk 1 above)
Comment (rosatravels) [reply zh-CN/Economics and Ethics of Private Property Studies in Political Economy and Philosophy/03_p01_ch02_01.md]: My appology.  I didn't read correctly.

**Pull request merged**